packages/x-components/src/x-modules/recommendations/components/recommendations.vue

```diff
@@ -161,43 +161,6 @@ recommendations.
 </Recommendations>
 ```
 
-### Play with props
```

Comment:
I think we could keep this example as `maxItemsToRender` is actually a prop of this component that is even being tested in the tests. 

Apart from changing `BaseSuggestions` to `Recommendations`, the prop `maxItemsToRender` needs to be corrected as it is misspelled as `maxItemToRender` in the example template. It's misspelled also in `PopularSearches`, `BaseSuggestions`, and `NextQueries` files.